src/ring_theory/witt_vector/domain.lean

```diff
@@ -0,0 +1,209 @@
+/-
+Copyright (c) 2022 Robert Y. Lewis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robert Y. Lewis
+-/
+
+import ring_theory.witt_vector.identities
+
+/-!
+
+# Witt vectors over a domain
+
+This file builds to the proof `witt_vector.domain`,
+an instance that says if `R` is an integral domain, then so is `𝕎 R`.
+Most of the file develops an API around iterated applications
+of `witt_vector.verschiebung` and `witt_vector.frobenius`.
+
+The [proof sketch](https://tinyurl.com/2p8cwrn7) goes as follows:
```

Comment:
```suggestion
The [proof sketch](https://math.stackexchange.com/questions/4117247/ring-of-witt-vectors-over-an-integral-domain/4118723#4118723)
goes as follows:
```

Reply:
... cool, I didn't realize there was an exception in the long lines check for this!